tool/bash/config_autocomplete.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+__gamer_check_gamer_info() {
+    # Check if the current directory is GAMER and the `configure.py` for generating `Makefile`
+    # $1 : configure.py filename
+
+    local isGitDir=`git rev-parse --is-inside-work-tree`
+    [ ! "$isGitDir" = true ] && return 1
+
+    local firstGAMERCommit=`git rev-list --max-parents=0 HEAD`
+    [ ! "$firstGAMERCommit" = "2705887745ac1fd254774c512af4a05b0f7f7fb6" ] && return 1
+
+    local configurePyFirstCommit=`git log --diff-filter=A -- $1 | head -n 1 | awk '{print $2}'`
+    [ ! "$configurePyFirstCommit" = "a93978a39b4388d169a0f45d1987f55486578fab" ] && return 1
+
+    return 0
+
+} # __gamer_check_gamer_info()
+
+
+__gamer_configure_autocomplete() {
+
+    local all_options sub_options all_option_array sub_option_array configure_filename
+    local not_set=true
+    local first=${COMP_WORDS[0]}
+    local second=${COMP_WORDS[1]}
+    local subsub="${COMP_WORDS[COMP_CWORD-2]}"
+    local sub="${COMP_WORDS[COMP_CWORD-1]}"
+    local cur="${COMP_WORDS[COMP_CWORD]}"
+
+    if [[ "$first" == "python"* ]]; then
+        configure_filename=$second
+    else
+        configure_filename=$first
+    fi
+
+    __gamer_check_gamer_info $configure_filename
+
+    # Not the GAEMR `configure.py`
+    if [ $? -ne 0 ]; then
+       COMPREPLY=( $( compgen -o default -- ${COMP_WORDS[COMP_CWORD]} ) )
+       # add a trailing space since `-o nospace`
+       for i in "${!COMPREPLY[@]}"; do
+           COMPREPLY[$i]="${COMPREPLY[$i]} "
+       done
+       return 0
+    fi
```

Comment:
4 spaces for indentation.
```suggestion
        COMPREPLY=( $( compgen -o default -- ${COMP_WORDS[COMP_CWORD]} ) )
        # add a trailing space since `-o nospace`
        for i in "${!COMPREPLY[@]}"; do
            COMPREPLY[$i]="${COMPREPLY[$i]} "
        done
        return 0
```